tests/integration/test_requirements_dev.py

```
@@ -257,14 +273,209 @@
+            content = f.read()
+        
+        assert len(content) > 0, "requirements-dev.txt is empty"
+        assert content.endswith(b'\n'), (
+            "requirements-dev.txt should end with a newline character (Unix convention)"
+        )
+    
+    def test_pyyaml_and_types_on_separate_lines(self):
+        """Test that PyYAML and types-PyYAML are on separate lines (not combined)."""
+        assert REQUIREMENTS_FILE.exists(), "requirements-dev.txt not found"
+        
+        with open(REQUIREMENTS_FILE, 'r') as f:
```

Ruff (0.14.6)

Unnecessary mode argument

Remove mode argument

(UP015)